### ISSUE TITLE
Cat facts API no longer available

### ISSUE THREAD

**Issue by hectorsector** [githubtraining/write-docker-actions#9]
@thinkverse mentioned in the [community forum](https://github.community/t/stuck-on-github-actions-write-docker-container-actions/129811) the cat facts API is no longer available. If that's the case, we should change the course to use a working API.

**Comment by brianamarie** [comment]
If anyone would like to contribute, our first step is duplicating this issue.

**Comment by thinkverse** [comment]
Would trying to enter the URL from the response file itself be a satisfactory replication? Or does the whole course need to be replicated to the point that it fails? 🤔

### Screenshot of application error on Cat API

![Screenshot of application error on Cat API](https://user-images.githubusercontent.com/2221746/94704252-031db180-0340-11eb-938f-e2d6b50bc31d.gif)

**Comment by brianamarie** [comment]
@thinkverse Good point - I think this can be considered duplicated, and now is a bug that is a breaking bug. I will move accordingly on the project board.

To fix this, I think we would want to find an alternative API, and update the course instructions accordingly.
